### DISCUSSION TITLE
Support for HP Z620 and HP 6200 Pro

### DISCUSSION THREAD

**Discussion by MikaelKW** [Rem0o/FanControl.Releases#2072] [Ideas]
I have three HP Z620 and one HP 6200 Pro Microtower that runs 24/7. 
In BIOS you can set a idle fan speed, where the fans will go at a specific RPM at idle, and from there go higher depending on load/temprature. 
For monotoring of fan speed i can use HP Performance Advisor.
My question is if it would be possible for this software to have full control of fan speed and fan curve for these two models, and other OEM models?

**Comment by Rem0o** [comment]
See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility

OEM models most of the time use proprietary stuff that is not interfaced for third party use.

**Reply by MikaelKW** [reply]
Thanks for the clarification. I will raise the question at LibreHardwareMonitor GitHub page.
